templates/microprofile-payara/files/.platform.template.yaml

```diff
@@ -0,0 +1,53 @@
+  # An image URI (either base64-encoded or a URL) representing the template.
+
+  # Additional notes displayed in the template's detail view.
+  # Each note object is displayed as a small section heading with content below. Supports limited HTML.
+  notes:
+    - heading: "Apps & Services"
+      content: "Java 8<br/>Maven<br/>Eclipse MicroProfile<br/>Payara Micro"
```

Comment:
Usually I've seen the lines after runtime be services (i.e. https://github.com/platformsh/template-drupal8/blob/16c6b5a7f563436c816cbb3862837bc7869c00eb/.platform.template.yaml#L39)